src/components/DatePicker/OverlayComponent/OverlayComponent.style.ts

```diff
@@ -0,0 +1,45 @@
+import { css, SerializedStyles } from '@emotion/core';
+import { Theme } from 'theme';
+import { flex } from '../../../theme/functions';
+
+export const overlayWrapperStyle = () => (theme: Theme): SerializedStyles => css`
+  ${flex};
+`;
+
+export const optionsWrapperStyle = () => (theme: Theme): SerializedStyles => css`
+  border-right: 1px solid #dfdfdf;
+`;
+
+export const optionStyle = ({ selected }: { selected?: boolean }) => (
+  theme: Theme
+): SerializedStyles => css`
+  white-space: nowrap;
+  padding: ${theme.spacing.md};
+  font-weight: ${selected ? 'bold' : 'initial'};
+  cursor: pointer;
+`;
+
+export const buttonsMonthsWrapperStyle = ({ isRangePicker }: { isRangePicker: boolean }) => (
+  theme: Theme
+): SerializedStyles => css`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  margin-bottom: ${isRangePicker && theme.spacing.md};
+`;
+export const monthsWrapperStyle = () => (theme: Theme): SerializedStyles => css`
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  z-index: 10;
```

Comment:
Reference in an issue maybe: 

Me need to have a central configuration about z-Indexes to avoid issues

Reply:
i agree for future use